src/ai/models.py

```diff
@@ -0,0 +1,187 @@
+"""This module implements OpenVINO related classes and functionalities."""
+
+from ai.openvino_model import OVModel
+from PytorchWildlife.models import detection as pw_detection
+from torchvision.transforms import InterpolationMode, transforms
+import torch
+
+
+txt_animalclasses = {
+    "fr": [
+        "blaireau",
+        "bouquetin",
+        "cerf",
+        "chamois",
+        "chat",
+        "chevre",
+        "chevreuil",
+        "chien",
+        "ecureuil",
+        "equide",
+        "genette",
+        "herisson",
+        "lagomorphe",
+        "loup",
+        "lynx",
+        "marmotte",
+        "micromammifere",
+        "mouflon",
+        "mouton",
+        "mustelide",
+        "oiseau",
+        "ours",
+        "ragondin",
+        "renard",
+        "sanglier",
+        "vache",
+    ],
+    "en": [
+        "badger",
+        "ibex",
+        "red deer",
+        "chamois",
+        "cat",
+        "goat",
+        "roe deer",
+        "dog",
+        "squirrel",
+        "equid",
+        "genet",
+        "hedgehog",
+        "lagomorph",
+        "wolf",
+        "lynx",
+        "marmot",
+        "micromammal",
+        "mouflon",
+        "sheep",
+        "mustelid",
+        "bird",
+        "bear",
+        "nutria",
+        "fox",
+        "wild boar",
+        "cow",
+    ],
+    "it": [
+        "tasso",
+        "stambecco",
+        "cervo",
+        "camoscio",
+        "gatto",
+        "capra",
+        "capriolo",
+        "cane",
+        "scoiattolo",
+        "equide",
+        "genet",
+        "riccio",
+        "lagomorfo",
+        "lupo",
+        "lince",
+        "marmotta",
+        "micromammifero",
+        "muflone",
+        "pecora",
+        "mustelide",
+        "uccello",
+        "orso",
+        "nutria",
+        "volpe",
+        "cinghiale",
+        "mucca",
+    ],
+    "de": [
+        "Dachs",
+        "Steinbock",
+        "Rothirsch",
+        "Gämse",
+        "Katze",
+        "Ziege",
+        "Rehwild",
+        "Hund",
+        "Eichhörnchen",
+        "Equiden",
+        "Ginsterkatze",
+        "Igel",
+        "Lagomorpha",
+        "Wolf",
+        "Luchs",
+        "Murmeltier",
+        "Kleinsäuger",
+        "Mufflon",
+        "Schaf",
+        "Mustelide",
+        "Vogen",
+        "Bär",
+        "Nutria",
+        "Fuchs",
+        "Wildschwein",
+        "Kuh",
+    ],
+}
+
+
+class OVMegaDetectorV5(pw_detection.MegaDetectorV5):
+    """MegaDetectorV5 class for detection model"""
+
+    def __init__(self, device):
```

Comment:
would it make sense to have a default to device param? E.g., device = 'cpu' or you prefer init to fail if device is not specified?

Reply:
I'd like to have only the top level file with the default device class, and the lower hierarchical ones (like the one in the ai package), that needs it. In that way, there is no ambiguity across multiple classes

Reply:
ok